src/services/auth/AuthService.js

```
@@ -165,15 +165,25 @@ class AuthService {
       // Create a new axios instance without interceptors
       const axiosInstance = axios.create();
 
+      // Get the access token
+      const accessToken = this.getAccessToken();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)